src/services/user/user-router.ts

```diff
@@ -128,7 +128,7 @@ userRouter.get("/:USERID", strongJwtVerification, async (req: Request, res: Resp
         if (userInfo) {
             return res.status(Constants.SUCCESS).send(userInfo);
         } else {
-            return res.status(Constants.INTERNAL_ERROR).send({ error: "UserNotFound" });
+            return res.status(Constants.NOT_FOUND).send({ error: "UserNotFound" });
```

Comment:
This shouldn't be 404 - 404 is reserved for API endpoint not found. Try 400 instead (invalid request/UserNotFound).

Reply:
Overruled